lib/mocha/mocha.ts

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+((context) => {
+  var Mocha = context['Mocha'];
+  
+  if (typeof Mocha === 'undefined') {
+    throw new Error('Missing Mocha.js');
+  }
+
+  if (typeof Zone === 'undefined') {
+    throw new Error('Missing Zone.js');
+  }
+
+  let ProxyZoneSpec = Zone['ProxyZoneSpec'];
+
+  if (!ProxyZoneSpec) {
+    throw new Error('Missing ProxyZoneSpec');
+  }
+
+  if(Mocha['__zone_patch__']){
+    throw new Error('"Mocha" has already been patched with "Zone".');
+  }
+
+  Mocha['__zone_patch__'] = true;
+
+  let rootZone = Zone.current;
+
+  function wrapCallbackInProxyZone(fn: Function): Function {
```

Comment:
Okey, I will try to fix that part.
Allright, yes I can have a look at that and merge our work.